app/auth/id-token/nonce.js

```diff
@@ -9,22 +9,15 @@ const generate = (request) => {
 }
 
 const verify = (request, idToken) => {
-  console.log(`${new Date().toISOString()} Verifying id_token nonce`)
-  try {
-    if (typeof idToken === 'undefined') {
-      throw new Error('Empty id_token')
-    }
-    const nonce = session.getToken(request, sessionKeys.tokens.nonce)
-    if (!nonce) {
-      throw new Error('HTTP Session contains no nonce')
-    }
-    if (nonce !== idToken.nonce) {
-      throw new Error('Nonce mismatch')
-    }
-  } catch (error) {
-    console.log(`${new Date().toISOString()} Error while verifying id_token nonce: ${error.message}`)
-    console.error(error)
-    throw error
+  if (typeof idToken === 'undefined') {
```

Comment:
Should this be typeof?

Reply:
Not something I changed, but yes it does seem cumbersome assertion when `if (idToken === undefined)` would also cover it.